examples/sessions/src/main.rs

```diff
@@ -17,10 +17,17 @@ use axum::{
     routing::get,
     AddExtensionLayer, Router,
 };
+use cookie::Cookie;
+use lazy_static::lazy_static;
 use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
 use std::net::SocketAddr;
 use uuid::Uuid;
 
+lazy_static! {
+    static ref SESSION_ID: String = Uuid::new_v4().to_string();
```

Comment:
What is this for?

Reply:
If the cookie's key is always the same, the program will assume that he have already set the cookie when the program runs multiple times.

To prevent this problem, I used uuid.